py-pkgs/08-ci-cd.ipynb

```diff
@@ -721,44 +719,37 @@
+    "PSR's [python-semantic-release/python-semantic-release@v8.3.0](https://python-semantic-release.readthedocs.io/en/latest/github-action.html) action will need to interact with our GitHub repository to modify our *`CHANGELOG.md`* file and bump the package version in `pyproject.toml`. To give PSR permission to do this there's a few things we need to do:\n",
     "\n",
-    "1. Provide PSR with a GitHub [access token](https://python-semantic-release.readthedocs.io/en/latest/envvars.html#env-gh-token) called `GH_TOKEN` to allow it to read/write files in our repository. At the start of each workflow run, [GitHub automatically creates](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) such a token for us, called `GITHUB_TOKEN`. We can pass this token to PSR using the `env` key and the syntax `${{ secrets.GITHUB_TOKEN }}`. You can read more about this syntax in the [GitHub documentation](https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow), but we've done it for you in the code shown below.\n",
-    "2. Configure the Git credentials of the runner machine by setting the username as `github-actions` and email address as `github-actions@github.com`. If you don't configure these credentials, the workflow won't be able to make changes to our repository, as you can read more about in the GitHub [documentation](https://docs.github.com/en/get-started/getting-started-with-git/setting-your-username-in-git). We've also done this for you in the code shown below, so all you need to do is add this code as the next step in your CD workflow file.\n",
+    "1. Give content write permissions to GitHub actions, we did that above by adding `permissions:` and then `content: write` below it in the `cd` job in `ci-cd.yml`.\n",
+    "2. Provide the first PSR GitHub action with a GitHub [access token](https://python-semantic-release.readthedocs.io/en/latest/envvars.html#env-gh-token) to allow it to read/write files in our repository. At the start of each workflow run, [GitHub automatically creates](https://docs.github.com/en/actions/security-guides/automatic-token-authentication) such a token for us, called `GITHUB_TOKEN`. We can pass this token to PSR using the `with` key and syntax `github_token: ${{ secrets.GITHUB_TOKEN }}`. We've done it for you in the code shown below.\n",
     "\n",
     "```yaml\n",
-    "    - name: Use Python Semantic Release to prepare release\n",
-    "      env:\n",
-    "        GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}\n",
-    "      run: |\n",
-    "          git config user.name github-actions\n",
-    "          git config user.email github-actions@github.com\n",
-    "          poetry run semantic-release publish\n",
+    "- name: Use Python Semantic Release to prepare release\n",
```

Comment:
lines from `id` down need to be de-indented, here's the current render:

![image](https://github.com/py-pkgs/py-pkgs/assets/40379073/4c10ad85-90b8-4d29-8b7b-8ea4493995b1)